### DISCUSSION TITLE
ERR_CONNECTION_REFUSED when trying to go to Spotify Web

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#5674] [Q&A] [answered]
When I try to go on [Spotify web](https://open.spotify.com/browse), I see the error

> This site can’t be reachedopen.spotify.com refused to connect.
> Try:
> 
> Checking the connection
> Checking the proxy and the firewall
> ERR_CONNECTION_REFUSED

is this something that I can solve by tweaking qutebrowser settings?

**Comment by The-Compiler** [comment, marked as answer]
Works fine for me. Does it work for you with another browser? What about a different network?

**Reply by Aster89** [reply]
Chromium gives the same result. So it has to do with the network... I'll see.
